src/content/docs/configuration/v4_migration.md

```diff
@@ -3,132 +3,143 @@ id: v4_migration
+**Please also read the Other Changes section** - there are a number of changes that are not just "move some config from one place to another". For example, in key mapping `<leader>` is no longer recognized.
 
-- We have removed Bufferline and are now using Heirline and `astronvim.utils.status` (previously was in `astronvim.status` but is now accessed with `require("astronvim.utils.status")`) for our own performant and custom tabline.
+If you get stuck, people on the [Discord](https://discord.astronvim.com/) forum are active and friendly! Like all humans, sometimes they are grumpy, so be nice to them! The best place to post is most likely the `#help-forum`, but poke around a few of the other channels, you never know what you will find that is useful.
 
-- `:AstroReload` has been removed. There are a couple reasons for this, it was never very reliable and hard to maintain and lazy.nvim strictly does not support hot reloading neovim configurations.
+### Migrating to AstroCore
 
-- The `require("astronvim.utils.status").component.macro_recording` status component has been removed. Please use the improved `require("astronvim.utils.status").component.cmd_info` component.
+🔗 [AstroCore example configuration](https://github.com/AstroNvim/astrocore/tree/main#%EF%B8%8F-configuration)
+📖 `:help astrocore`
 
-- `lsp.server-settings` has been renamed to `lsp.config`. If you have the `["server-settings"]` table in your `user/init.lua` file, just rename it to `config`. If you have the folder `user/lsp/server-settings`, just rename the folder to `user/lsp/config`.
+The following version 3 `user/init.lua` configuration keys move to the same keys in the version 4 `lua/plugins/astrocore.lua` template file:
 
-- `luasnip` options table has been removed. Please see the updated [Custom Snippets Documentation](../../recipes/snippets) for the new way to extend the default configuration of LuaSnip to add new loaders.
+- `autocmds`
+- `mappings`
+- `options`
 
-- `which-key` options table has been removed. Which-key menu titles can now be easily added into the `mappings` table by setting a binding to a table with the `name` property set and it will be passed to `which-key`. For example, you can add the following to the `mappings` table: `["<Leader>b"] = { name = "Buffer" }`.
+Additional non version 3 keys configured in the `lua/plugins/astrocore.lua` file are:
 
-- `nvim-autopairs` options table has been removed. Please see the updated [Customize Autopairs Documentation](../../recipes/autopairs) for the new way to extend the default configuration of autopairs and adding more rules.
+- `commands`
+- `features`
+- `git_worktrees`
```

Comment:
maybe worth noting that some of the features used to be global `vim.g` variables also `git_worktrees` also used to be a global `vim.g` variable

Reply:
I search vim help for all of the features and none matched. I'll let you document that bit.